packages/dx-react-grid/src/components/table-layout/virtual-table-layout.jsx

```diff
@@ -104,9 +102,8 @@ export class VirtualTableLayout extends React.PureComponent {
     }
 
     if (this.viewportTop !== node.scrollTop) {
-      this.viewportTop = node.scrollTop;
       this.setState({
-        visibleBodyRows: this.getVisibleRows(),
+        viewportTop: node.scrollTop,
```

Comment:
Can we extract `scrollTop` from `node`? It's using a lot of times.

Reply:
Yes, we can, but I am not sure that we should. Seems like function will become more verbose.